forms/core/Application.php

```diff
@@ -312,6 +266,10 @@ public static function update()
             $formTypeMap['simple_form'] = FormType::EMBEDDED;
 
             $clients = Application::$CampaignMonitor->get_clients();
+            if (isset($clients->error)) {
+                // TODO: handle error when it gets to this point

```

Comment:
Could we handle it now rather than leaving it as to do? May be simply logging if we expect that this is very unlikely to happen?